src/GLM.jl

```diff
@@ -14,6 +14,7 @@ module GLM
     import Base: (\), cholfact, convert, cor, show, size
     import StatsBase: coef, coeftable, confint, deviance, nulldeviance, dof, dof_residual, loglikelihood, nullloglikelihood, nobs, stderr, vcov, residuals, predict, fit, model_response, r2, r², adjr2, adjr², PValue
     import StatsFuns: xlogy
+    import StatsBase: FrequencyWeights, ProbabilityWeights, AnalyticWeights
```

Comment:
You'll need to bump the lower bound on StatsBase in the REQUIRE file to the version where these types were introduced.